### ISSUE TITLE
[java]InstantiationToGetClass can not detect the case which is written in two lines

### ISSUE THREAD

**Issue by wuchiuwong** [pmd/pmd#2539]
**Affects PMD Version:** 
6.22.0

**Rule:**
InstantiationToGetClass

**Description:**
InstantiationToGetClass can not detect the case which is written in two lines: <code>Foo foo = new Foo(); Class c = foo.getClass()</code>
This rule is implemented through xpath search:
```
//PrimarySuffix
 [@Image='getClass']
 [parent::PrimaryExpression
  [PrimaryPrefix/AllocationExpression]
  [count(PrimarySuffix) = 2]
 ]
```
which can only detect the case in single line: <code>Class c = new Foo().getClass()</code>

**Code Sample demonstrating the issue:**

```
Foo foo = new Foo();
Class c = foo.getClass()
```

**Expected outcome:**
false-negative

**Running PMD through:**
CLI
